vnext/Common/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="boost" version="1.68.0.0" targetFramework="native" />
```

Comment:
><package id="boost" version="1.68.0.0" targetFramework="native" /> [](start = 2, length = 66)

Can we prevent this from getting out of sync with the package dependency in folly/packages.config? Does this interact with the work @JunielKatarn is doing?